python_modules/dagster-graphql/dagster_graphql/implementation/asset_checks_loader.py

```diff
@@ -169,128 +172,3 @@ def get_checks_for_asset(
         )
 
         return self._checks[asset_key]
-
-
-def _execution_targets_latest_materialization(
```

Comment:
moved this (with some renames / new comments) to the body of AssetCheckExecutionRecord